<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomWidgetsDialog</class>
 <widget class="QDialog" name="CustomWidgetsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomWidgetsDialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1251</width>
     <height>691</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>进度条</string>
    </attribute>
    <widget class="QFlowProgressBar" name="label_circle1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>1181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFlowProgressBar" name="label_circle2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>1181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFlowProgressBar" name="label_rectangle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>1181</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFlowProgressTab" name="label_tabstyle">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>1181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>620</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Prev">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>620</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>图片导航</string>
    </attribute>
    <widget class="QPictureSlides" name="label_pictures">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>1161</width>
       <height>541</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_auto">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>630</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>自动换图</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>630</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>630</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Toast</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>731</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Toast</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>弹出位置:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>弹出时长:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_pos">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_time">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_show">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>270</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>显示内容：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_toast">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>441</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>721</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>卡片对话框</string>
     </property>
     <widget class="QPushButton" name="pushButton_card">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>30</y>
        <width>321</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>显示卡片对话框</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gallery</string>
    </attribute>
    <widget class="QGallery" name="gallery_main" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>1111</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_nextStep">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>630</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_prev">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>630</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QFlowProgressBar</class>
   <extends>QLabel</extends>
   <header>CustomWidgets\QFlowProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>QFlowProgressTab</class>
   <extends>QLabel</extends>
   <header>CustomWidgets\QFlowProgressTab.h</header>
  </customwidget>
  <customwidget>
   <class>QPictureSlides</class>
   <extends>QLabel</extends>
   <header>CustomWidgets\QPictureSlides.h</header>
  </customwidget>
  <customwidget>
   <class>QGallery</class>
   <extends>QWidget</extends>
   <header>CustomWidgets\QGallery.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
